keylime/ima/ima_dm.py

```diff
@@ -395,6 +401,8 @@ def _check_attr(attr: Optional[Union[int, str, bool]], reference_value: Optional
     if attr is None and reference_value is not None:
         return False
 
+    assert attr is not None
```

Comment:
We can just drop the `and reference_value is not None` from line 401 right? Because that is we already returned if the value was None.

Reply:
good point: 07ef85c